meetups/2024/2024-11-28-talk-night/program.md

```diff
@@ -0,0 +1,61 @@
+# Rust Aarhus meetup at [Lind Capital][lc]
+
+Date: 2024-11-28
+
+** PLEASE NOTICE THAT WE ARE LOOKING FOR A SHORT TALK OF 10-20 MINUTES. SHARE YOU LATEST DISCOVERY OR TALK ABOUT THE FUN PROJECT YOU ARE WORKING ON. PLEASE CONTACT THE ORGANIZER FOR MORE DETAILS. **
```

Comment:
```suggestion
> [!IMPORTANT] 
> **PLEASE NOTICE** THAT WE ARE LOOKING FOR A SHORT TALK OF 10-20 MINUTES.
>
> SHARE YOUR LATEST DISCOVERY OR TALK ABOUT THE FUN PROJECT YOU ARE WORKING ON.
>
> ***PLEASE CONTACT THE ORGANIZER FOR MORE DETAILS***
```

Using the [GFM alerts](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts) syntax

Reply:
Thanks for the suggestion, but since very few people are looking at the program in GitHub as it is copied/pasted to Meetup, I will leave this one out.